test/e2e/features/story_health.feature

```diff
@@ -41,16 +41,13 @@ Feature: End-to-end health check
             """
         And stdout should contain "You can add applications to this project with the 'new-app' command."
 
-    @darwin @linux @windows
+    @darwin @linux @windows @testdata
     Scenario: Create and test app
-        When executing "oc create deployment httpd-example --image=quay.io/bitnami/nginx --port=8080" succeeds
+        When executing "oc create -f httpd-deployment.yaml" succeeds
         Then stdout should contain "deployment.apps/httpd-example created"
         When executing "oc rollout status deployment httpd-example" succeeds
         Then stdout should contain "successfully rolled out"
-        When executing "oc expose deployment httpd-example --port 8080" succeeds
-        Then stdout should contain "httpd-example exposed"
-        When executing "oc expose svc httpd-example" succeeds
-        Then stdout should contain "httpd-example exposed"
+        And with up to "2" retries with wait period of "60s" http response from "http://httpd-example-testproj.apps-crc.testing" has status code "200"
```

Comment:
See: https://github.com/code-ready/crc/pull/3219/files#r897518410. Can content be checked? How easy or difficult would this be?

Reply:
I think something as simple as this would work: 

```
When executing "curl http://httpd-example-testproj.apps-crc.testing" succeeds
Then stdout should contain "Hello CRC"
```
The string is reasonably unique, so that should be OK. Not sure how much value this additional check brings, but it's basically for free so it doesn't matter I guess.

Reply:
> I think something as simple as this would work:
```
When executing "curl http://httpd-example-testproj.apps-crc.testing" succeeds
Then stdout should contain "Hello CRC"
```

I suspect the retry is needed in case the pod is not fully up yet when we start trying curl, in which case it's not going to be as simple.

Reply:
In that case the following might work out of the box
```
Then with up to "20" retries with wait period of "30s" command "curl http://httpd-example-testproj.apps-crc.testing" output matches ".*Hello CRC.*"
```
Not saying it needs to go in, just that some templates exist for these things. But there could be other issues that I'm missing and why it won't work.

Reply:
Tried what you suggested, and a few variations around it, but for some reason, `Hello World` on stdout is not captured correctly even if it shows up as expected in the logs.

Reply:
Hmmm. Good investigation, I don't think I've seen this before... Thanks